dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -266,6 +266,8 @@ echo "done!"
 # If command == clean up then delete all openshift objects
 # -------------------------------------------------------------
 if [ "${COMMAND}" == "cleanup" ]; then
+  echo "[CHE] Stopping the Che server..."
+  oc scale --replicas=0 --timeout=3h dc che
```

Comment:
as a side note, do we really want to wait 3h there ?

Reply:
probably not :-) The idea though is to give enough time to stop all the workspaces started on the various external clusters / namespaces associated to users in the multi-tenant scenario.

Reply:
@l0rd What value would you give to this timeout ?

Reply:
3 minutes?

Reply:
Fixed in commit cc969613ece31042561699261281c7813498dc0d